src/test/scala/sangria/macros/derive/DeriveObjectTypeMacroSpec.scala

```diff
@@ -202,6 +202,19 @@ class DeriveObjectTypeMacroSpec extends WordSpec with Matchers with FutureResult
       tpe.fields(2).fieldType should be (IntType)
     }
 
+    "allow field names transformation" in {
+      val transformer1 = (s: String) => s.toUpperCase
+      val tpe = deriveObjectType[Unit, TestSubjectAnnotated](TransformFieldNames(transformer1))
+      tpe.fields.forall(f => f.name == transformer1(f.name)) shouldBe true
```

Comment:
If I understand correctly, `f.name` is already transform name. This assertion feels a bit unsafe for me. What do you think about explicitly asserting on every field? E.g.

```scala
tpe.fields.map(_.name) should (
  have(size(2)) and
  contain("ID") and 
  contain("MYLIST"))
```

For test assertions I prefer this more explicit style. It makes it easier for a reader to see the expected values.